app/src/main/java/net/squanchy/tweets/view/TweetItemView.java

```diff
@@ -1,41 +1,71 @@
 package net.squanchy.tweets.view;
 
+import android.app.Activity;
 import android.content.Context;
+import android.support.annotation.Nullable;
 import android.text.method.LinkMovementMethod;
 import android.util.AttributeSet;
+import android.widget.ImageView;
 import android.widget.TextView;
 
 import net.squanchy.R;
+import net.squanchy.imageloader.ImageLoader;
+import net.squanchy.imageloader.ImageLoaderInjector;
+import net.squanchy.support.ContextUnwrapper;
+import net.squanchy.support.lang.Optional;
 import net.squanchy.support.widget.CardLayout;
-import net.squanchy.tweets.domain.view.Tweet;
-import net.squanchy.tweets.util.TweetFormatter;
+import net.squanchy.tweets.domain.view.TweetViewModel;
 import net.squanchy.tweets.util.TwitterFooterFormatter;
 
 public class TweetItemView extends CardLayout {
 
-    private TextView tweetText;
-    private TweetFooterView tweetFooter;
+    @Nullable
+    private ImageLoader imageLoader;
```

Comment:
Hm I thought about that but I don't know if we can do it with sourcesets instead of doing it passing booleans around. If it's the latter I'd rather have it inside the view